example/gluon/data.py

```diff
@@ -109,18 +111,17 @@ def transform(image, label):
         # resize the shorter edge to 224, the longer edge will be greater or equal to 224
         resized = mx.image.resize_short(image, 224)
         # center and crop an area of size (224,224)
-        cropped, crop_info = mx.image.center_crop(resized, 224)
+        cropped, crop_info = mx.image.center_crop(resized, (224, 224))
         # transpose the channels to be (3,224,224)
-        transposed = nd.transpose(cropped, (2, 0, 1))
-        image = mx.nd.cast(image, dtype)
-        return image, label
+        transposed = mx.nd.transpose(cropped, (2, 0, 1))
```

Comment:
is dtype casting no longer necessary?

Reply:
Not in this script as it does astype() later. And that will also be more general for any dataset iterator